components/DBTableMocker.php

```diff
@@ -18,4 +32,23 @@ public function getTableDescription(string $tableName):array {
 
             return $columns;
         }
+
+        /**
+         * Sets the current column
+         *
+         * @param string $column
+         * @return void
+         */
+        public function column(string $column):void {
+            $this->currentColumn = $column;
+        }
```

Comment:
This method sets the column which is being changed. The column might be given a key(primary, unique etc.) or it can be made auto_increment.